Makefile

```diff
@@ -107,7 +107,7 @@ clean-images clean-enterprise-gateway-demo clean-nb2kg clean-yarn-spark clean-ke
 publish-images: ## Push docker images to docker hub
 	make WHEEL_FILE=$(WHEEL_FILE) VERSION=$(VERSION) -C etc $@
 
-ENTERPRISE_GATEWAY_TAG?=dev
+ENTERPRISE_GATEWAY_TAG?=$(VERSION)
```

Comment:
We could do something like in [this makefile](https://github.com/apache/incubator-toree/blob/master/Makefile), to use dev tag when version contains .dev and use a specific version when it's a release version (without tag) ?

Reply:
Is ENTERPRISE_GATEWAY_TAG used any other than the itest-yarn-prep target? seems like the its the only place its used..if so then I think we should just remove it and use whats in VERSION. 

if so, did we want to use just “dev”?  it seems not very informative if someone were to develop on multiple versions simultaneously(not likely but maybe). having something like 2.0.0-dev would be more helpful...or we would adopt toree’s release nomenclature and use a pseudo-maven SNAPSHOT reference?

Reply:
I don't think that specific tag is necessary.  +1 for removal.

Reply:
+1 to removal